deploy/platform/kubernetes/templates/feature-rover/resources.yaml

```diff
@@ -57,13 +60,21 @@ spec:
             - name: ROVER_BACKEND_ADDR
               value: {{ template "skywalking.collector.address" . }}
             - name: ROVER_PROCESS_DISCOVERY_KUBERNETES_ANALYZER_ISTIO_ENVOY_SERVICE_NAME
-              value: mesh-svr::{{"{{"}}.Pod.LabelValue "service.istio.io/canonical-name,app.kubernetes.io/name,app"{{"}}"}}.{{"{{"}}.Pod.Namespace{{"}}"}}
+              value: mesh-svr::{{"{{"}}.Pod.LabelValue "service.istio.io/canonical-name,app.kubernetes.io/name,app" ""{{"}}"}}.{{"{{"}}.Pod.Namespace{{"}}"}}
             - name: ROVER_PROCESS_DISCOVERY_KUBERNETES_ANALYZER_ISTIO_APPLICATION_SERVICE_NAME
-              value: mesh-svr::{{"{{"}}.Pod.LabelValue "service.istio.io/canonical-name,app.kubernetes.io/name,app"{{"}}"}}.{{"{{"}}.Pod.Namespace{{"}}"}}
+              value: mesh-svr::{{"{{"}}.Pod.LabelValue "service.istio.io/canonical-name,app.kubernetes.io/name,app" ""{{"}}"}}.{{"{{"}}.Pod.Namespace{{"}}"}}
             - name: ROVER_PROCESS_DISCOVERY_KUBERNETES_ANALYZER_K8S_SERVICE_NAME
               value: skywalking-showcase::{{"{{"}}.Pod.ServiceName{{"}}"}}.{{"{{"}}.Pod.Namespace{{"}}"}}
             - name: ROVER_HOST_MAPPING
               value: /host
+            - name: ROVER_CORE_CLUSTER_NAME
+              value: skywalking-showcase
+            - name: ROVER_ACCESS_LOG_ACTIVE
+              value: "true"
+            - name: ROVER_ACCESS_LOG_EXCLUDE_NAMESPACES
+              value: gmp-system,kube-system,skywalking-showcase
```

Comment:
The namespace `skywalking-showcase` should not be hardcoded

Reply:
It's configurable when installing showcase like `make deploy.kubernetes NAMESPACE=demo`

Reply:
@mrproliu Could you follow this env variable?